lib/saml/user.rb

```diff
@@ -79,13 +79,11 @@ def authn_context
 
     def user_attributes_class
       case authn_context
-      when 'myhealthevet'; then SAML::UserAttributes::MHV
-      when 'dslogon'; then SAML::UserAttributes::DSLogon
-      when 'myhealthevet_multifactor', 'dslogon_multifactor', 'multifactor'
-        SAML::UserAttributes::IdMe
-      when 'dslogon_loa3', 'myhealthevet_loa3', LOA::IDME_LOA3
-        SAML::UserAttributes::IdMe
-      when LOA::IDME_LOA1
+      when 'myhealthevet', 'myhealthevet_multifactor'
+        SAML::UserAttributes::MHV
+      when 'dslogon', 'dslogon_multifactor'
```

Comment:
same comment here... 'dslogon' includes attributes from both ID.me and DS Logon, but `dslogon_multifactor` is identical to `multifactor`, it's only ID.me attributes and should not be treated as a DS Logon type of user attributes class.